report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -2,34 +2,46 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
-However, our objective is to develop models that demonstrate high accuracy and robustness, even on entirely unseen data.
-To achieve this, we employ k-fold cross-validation to ensure our models have high generalizability, thereby increasing the likelihood that they will perform as expected on new data.
+For this reason, it is common to use a validation set to tune hyperparameters and evaluate model performance during experimentation.
+However, further splitting the training set into a validation set exacerbates the challenges of limited data availability, as described in Section~\ref{subsec:challenges}.
+Our objective is to develop models that demonstrate high accuracy and robustness, even on entirely unseen data.
+To achieve this, we employ $k$-fold cross-validation.
+This allows us to train our models on $k-1$ folds and evaluate them on the remaining fold, which provides a more robust estimate of model performance.
+Since our approach includes large-scale optimization, we prefer $k$-fold cross-validation over \gls{loocv}.
+\gls{loocv} uses each individual sample as a single test case, resulting in $n$ iterations, where $n$ is the number of samples.
+In each iteration, models and preprocessors would need to be refit, making this approach too computationally expensive and time-consuming for the scope of our study.
 
-We use an $80\%/20\%$ split for training and testing sets. The training set is further subdivided into $k$ folds, which are used for cross validation.
-
-While we employ conventional techniques like holdout sets and k-fold cross validation, \gls{libs} data imposes additional challenges to the process.
+While we employ conventional techniques like holdout sets and k-fold cross validation, the dataset we use imposes additional challenges to the process.
 
 One of the primary challenges is preventing data leakage.
 As per concentration matrix $\mathbf{C}$ in Section~\ref{sec:problem_definition}, each target only has one ground truth concentration value per oxide.
 However, each target is shot at multiple locations, resulting in multiple instances of the same target in the dataset, as shown in Table~\ref{tab:final_dataset_example}.
-Although the intensity values vary for each location, they fundamentally represent different measurements of the same target.
+Although the intensity values vary for each location, they fundamentally represent measurements of the same target.
 If we were to randomly split the dataset, some locations from a target could end up in the testing set while others remain in the training set.
 This would cause data leakage, as the testing set would no longer consist solely of unseen targets.
 To prevent this, we ensure that each target is represented only once in the dataset by grouping data from all locations on a given target.
 
 Furthermore, the limited availability of data, as mentioned in Section~\ref{sec:problem_definition}, poses another significant challenge due to the difficult collection process.
```

Comment:
I see what you are trying to get at with the difficult collection process, but I feel like the way its phrased makes it seem like you want to talk about the collection process, but then you go into describing the size of the dataset without really connecting it to this point, which is: The collection process is difficult - Therefore we only have 408 samples. The fact that we only have such few samples means distribute the extreme values properly.

Reply:
I just removed the part about the collection process. Thoughts?
And since we mention the limited data already on line 13, we can also remove the extra `as mentioned in`.

```diff
- Furthermore, the limited availability of data, as mentioned in Section~\ref{sec:problem_definition}, poses another significant challenge due to the difficult collection process.
+ Furthermore, the limited availability of data poses another significant challenge.
```

I pushed this. Thoughts?